Signal/src/ViewControllers/RegistrationViewController.m

```diff
@@ -61,6 +61,9 @@ - (void)viewDidLoad {
 - (void)createViews
 {
     self.view.backgroundColor = [UIColor ows_signalBrandBlueColor];
+    self.view.userInteractionEnabled = YES;
+    [self.view
+        addGestureRecognizer:[[UITapGestureRecognizer alloc] initWithTarget:self action:@selector(backgroundTapped:)]];
```

Comment:
On "verification code" view, tapping the background _deselects_ the text field, since there's a ton of cramped UI in that view and it might help access/view the bottom elements in some languages.  

On "registration" view we want to do the opposite since its less cramped - errant taps should select the text field, to help nudge the users through the registration process.